spec/unit/job_fetcher_spec.rb

```diff
@@ -1,3 +1,5 @@
+require 'spec_helper'
+
```

Comment:
I think this should be done by adding `-r spec_helper` to `rspec` parameters. Otherwise we better go and add this to all spec files and stop adding `-r`. For consistency I mean. I think it should either be required in the file or on command line but not some mix.

btw it is documented in https://github.com/3scale/apisonator/blob/master/DEVELOPMENT.md#running-tests-advanced

Reply:
I get your point. IMO we should require it from code for all specs, that way we can run the tests out of the  box from Ruby Mine and also from console or `rake`. There's probably one single point during tests initialization where all tests pass through and we can just `require spec_helper` from there.

Reply:
OK. It seems there's not such single point. I'll just remove the require from all tests. When we need to launch a particular test from RubyMine, we'll have to manually add `-r rspec_helper` to the launcher.

Reply:
If you edit the `rspec` template in idea, any automatically created run configurations inherit it.

Reply:
I'm OK either way. If `-r spec_helper` is already a documented way to run the tests, let's stick with it.